Funcky.Test/Monads/OptionTest.cs

```diff
@@ -221,8 +221,8 @@ public void GivenAnOptionAndAndAFuncToOptionItShouldBeFlattened()
             var none = Option<int>.None;
             var some = Option.Some(42);
 
-            Assert.False(none.AndThen(_ => Option.Some(1337)).Match(false, v => v == 1337));
-            Assert.True(some.AndThen(_ => Option.Some(1337)).Match(false, v => v == 1337));
+            Assert.False(none.AndThen(_ => Option.Some(1337)).Match(none: false, some: v => v == 1337));
```

Comment:
I think this should be replaced with `FunctionalAssert`

Reply:
Improved in 4d132c9802ee79cac51fafb25adccaa27fdf7843